From 456a58e4cb06c9cbdaadfc3d2e54ef21ec8405fc Mon Sep 17 00:00:00 2001
From: Yogesh Wani <yogesh.wani@arm.com>
Date: Wed, 7 May 2025 16:51:27 +0000
Subject: [PATCH] Remove duplicate configuration parameters for corstone-1000

The PS_NUM_ASSET is duplicated in the cmake.config and the
config_tfm_target.h file under corstone-1000. The commit removes
the one from the cmake.config and keeps the one in the header file.

The whole rationale behind this is for the vendor to be able
to override the configuration using the cmake file.

Signed-off-by: Yogesh Wani <yogesh.wani@arm.com>
Upstream-Status: Backport [948cb8e7601dcf1fe822d855c77749287fe6d9bd]
---
 platform/ext/target/arm/corstone1000/config.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/platform/ext/target/arm/corstone1000/config.cmake b/platform/ext/target/arm/corstone1000/config.cmake
index 6a805a1220..708bacaedf 100644
--- a/platform/ext/target/arm/corstone1000/config.cmake
+++ b/platform/ext/target/arm/corstone1000/config.cmake
@@ -68,4 +68,3 @@ endif()
 # Platform-specific configurations
 set(CONFIG_TFM_USE_TRUSTZONE            OFF)
 set(TFM_MULTI_CORE_TOPOLOGY             ON)
-set(PS_NUM_ASSETS                       "40"        CACHE STRING    "The maximum number of assets to be stored in the Protected Storage area")
--
2.34.1

